src/aui/auibook.cpp

```diff
@@ -3446,6 +3446,127 @@ bool wxAuiNotebook::InsertPage(size_t index, wxWindow *page,
     }
 }
 
+class wxAuiLayoutObject
+{
+public:
+    wxAuiLayoutObject(wxSize &s, unsigned short l, unsigned short dir, unsigned short r, unsigned short pos)
+    {
+        size = s;
+        layer = l;
+        direction = dir;
+        switch (dir)
+        {
+            case wxAUI_DOCK_CENTER: direction = 0; break;
+            case wxAUI_DOCK_LEFT:   direction = 1; break;
+            case wxAUI_DOCK_RIGHT:  direction = 2; break;
+            case wxAUI_DOCK_TOP:    direction = 3; break;
+            case wxAUI_DOCK_BOTTOM: direction = 4; break;
+            default:                direction = 5;
+        }
+        row = r;
+        position = pos;
+    }
+    void MergeLayout(wxAuiLayoutObject *lo2)
```

Comment:
This method should take const reference and not a non-const pointer as its argument is not supposed to be null and is not modified by it.

Reply:
I corrected this.